app/services/search_service.rb

```diff
@@ -29,7 +29,7 @@ def all(threads: true, searchers: Settings.ENABLED_SEARCHERS)
     searches
   end
 
-  def one(searcher, timeout: 30)
+  def one(searcher, query, timeout: 30)
```

Comment:
Isn't this `query` the same as the query passed in to the initializer?  If so, I think a better change is to drop it from the calling arguments.

Reply:
makes sense; made that change and removed from the caller in `SearchService#all`